website/content/docs/provisioners/shell.mdx

```diff
@@ -1,22 +1,18 @@
 ---
 description: |
-  The shell Packer provisioner provisions machines built by Packer using shell
-  scripts. Shell provisioning is the easiest way to get software installed and
-  configured on a machine.
-page_title: Shell - Provisioners
+  The `shell` Packer provisioner provisions machines built by Packer using shell
+  scripts. Use the `shell` provisioner install and configure software on a machine.
```

Comment:
```suggestion
  The `shell` Packer provisioner runs shell scripts on the machine Packer builds. Use the `shell` provisioner to install and configure software on a machine.
```

What do you think about this tweak to the wording? "Provisions machines using shell scripts" feels a little ambiguous to me, but maybe that's just me :)